docs/services/slam/cartographer/_index.md

```diff
@@ -0,0 +1,364 @@
+| `config_params` |  map[string] string | Optional | Parameters available to fine tune the `cartographer` algorithm: [read more below](#config_params). |
+
+{{% alert title="Caution" color="caution" %}}
+
+- If `use_live_data: true`, `delete_processed_data: true` by default.
+- If `use_live_data: false`, `delete_processed_data: false` by default.
+
+Setting `delete_processed_data: true` and `use_live_data: false` is invalid and will result in an error.
+{{% /alert %}}
+
+### `config_params`
+
+Adjust these parameters to fine-tune the algorithm `cartographer` utilizes in aspects like submap size, mapping update rate, and feature matching details:
+
+| Parameter Mode | Description | Inclusion | Default Value | Notes |
+| -------------- | ----------- | --------- | ------------- | ----- |
+| `mode` | `2d` | **Required** | None | |
+| `optimize_every_n_nodes` | How many trajectory nodes are inserted before the global optimization is run. | Optional | `3` | |
+| `num_range_data` | Number of measurements in each submap. | Optional | `100` | |
+| `missing_data_ray_length` | Replaces the length of ranges that are further than `max_range` with this value. | Optional | `25` | Nominally set to max length. |
+| `max_range` | Maximum range of valid measurements. | Optional | `25` | |
+| `min_range` | Minimum range of valid measurements. | Optional | `0.2` | |
+| `max_submaps_to_keep` | Number of submaps to use and track for localization. | Optional | `3` | Only for [LOCALIZING mode](#mapping-modes). |
+| `fresh_submaps_count` | Length of submap history considered when running SLAM in updating mode. | Optional | `3` | Only for [UPDATING mode](#mapping-modes). |
+| `min_covered_area` | The minimum overlapping area, in square meters, for an old submap to be considered for deletion. | Optional | `1.0` | Only for [UPDATING mode](#mapping-modes). |
+| `min_added_submaps_count` | The minimum number of added submaps before deletion of the old submap is considered. | Optional | `1` | Only for [UPDATING mode](#mapping-modes). |
+| `occupied_space_weight` | Emphasis to put on scanned data points between measurements. | Optional | `20.0` | |
+| `translation_weight` | Emphasis to put on expected translational change from pose extrapolator data between measurements. | Optional | `10.0` | |
+| `rotation_weight` | Emphasis to put on expected rotational change from pose extrapolator data between measurements. | Optional | `1.0` | |
+
+## Troubleshooting
+
+### Tip: Lidar Mounting for the Viam Rover
+
+If you have a Viam Rover and need a mount for your RPLidar, you can 3D print an adapter plate.
+The STL file for an adapter plate is available on [GitHub](https://github.com/viamrobotics/VR1-22-A001/blob/master/CAD/RPIidar_adapter.STL).
+
+### Known Issues
+
+#### Maps not appearing in UI
+
+When generating a larger map, the Cartographer service can take a while to return and visualize the map.
```

Comment:
```suggestion
When generating a larger map, it will take longer for Cartographer to return the desired map.
This can result in errors or failed requests for a map; however, this will not affect the `viam-server` or `cartographer` process.
Re-requesting the map can and should be successful, although there is currently a fundamental limit for the size of map that can be transmitted to the UI and this issue will become more common as you approach it.
```